tools/projmgr/docs/Manual/Overview.md

```diff
@@ -102,7 +102,7 @@ The CMSIS Pack repository must be present in the development environment.
 - Before running `csolution` the location of the pack repository shall be set via the environment variable
 `CMSIS_PACK_ROOT` otherwise its default location (todo what is the default?) will be taken.
```

Comment:
The default depends on the platform and is given by this function:
https://github.com/Open-CMSIS-Pack/devtools/blob/main/libs/crossplatform/src/CrossPlatformUtils.cpp#L34-L49